xml/mysql_banners.xml

```diff
@@ -175,6 +186,8 @@
     <param pos="0" name="os.family" value="Linux"/>
     <param pos="0" name="os.product" value="Linux"/>
     <param pos="0" name="os.version" value="12.04"/>
+    <param pos="0" name="os.certainty" value="0.80"/>
+    <param pos="0" name="os.device" value="General"/>
```

Comment:
Sure, I can remove these. I was trying to keep it in line with (most) desktop/server OS fingerprints in Nexpose.